NoMansLand/Assets/Scripts/Player/CharacterControllerMovement.cs

```diff
@@ -14,14 +14,20 @@ public class CharacterControllerMovement : MonoBehaviour
 
     // Move
     private Vector3 move;
-    private float speed = 85f;
+    private float speed = 60f;
```

Comment:
Can we keep the regular speed the same? feels slow in normal gameplay and the aliens are faster than you as well